### ISSUE TITLE
Present IDE from calling WorldMorph::fillPage

### ISSUE THREAD

**Issue by cycomachead** [jmoenig/Snap--Build-Your-Own-Blocks#1164]
I'd like to have some text / UI around the Snap! canvas, and am trying to use `isAutoFill = false` but the IDE is still making a call to `fillPage()` which means that Snap<em>!</em> still fills up the window. 

Is there a way to prevent the window filling from happening without monkey patching? I guess what I really want is probably for the IDE to autofill, but _not_ the world, right?

**Comment by cycomachead** [comment]
Ah ha! So I can pass `false` to the `world` when initializing that, but now I can't see a way to make the canvas fill the width of its container element?

**Comment by jmoenig** [comment]
that functionality may not even yet exist, but it should be easy to add...

**Comment by cycomachead** [comment]
Yeah, so far I can hack around this -- which isn't too hard, but getting the `onResize` stuff correct is a bit weird right now.

**Comment by DavidGasku** [comment]
@cycomachead : did you get anywhere with this? I'm going to try to have a fillParent flag in adition to fillPage, so it works also onResize. Just checking that is already done in some fork/branch.

**Comment by cycomachead** [comment]
No. Not ever fully working. I think I got close on this app but it’s now a few years old and not very generic. 

https://github.com/snap-cloud/lambda/blob/master/app/views/questions/_snap_autograder.html.erb#L31
